index.bs

```diff
@@ -1305,8 +1309,10 @@ Create a named {{MLOperand}} based on a descriptor, that can be used as an input
   <summary>
     The <dfn method for=MLGraphBuilder>input(|name|, |descriptor|)</dfn> method steps are:
   </summary>
+    1. If [=this=].{{MLGraphBuilder/[[hasBuilt]]}} is true, then [=exception/throw=] an "{{InvalidStateError}}" {{DOMException}}.
     1. If |name| is empty, then [=exception/throw=] a {{TypeError}}.
     1. If [=MLOperandDescriptor/checking dimensions=] given |descriptor| returns false, then [=exception/throw=] a {{TypeError}}.
+    1. If any {{MLOperand}}s in [=this=]'s [=MLGraphBuilder/graph=]'s [=computational graph/inputs=] have the same {{MLOperand/[[name]]}}, then [=exception/throw=] a {{TypeError}}.
```

Comment:
nit: I'd move this step up one, so the checks related to `name` are adjacent.

Reply:
nit: "have the same name" ... as what? It's pretty obvious in context, but could be made more rigorous as "have name equal to name"